YHP_website/src/main/webapp/WEB-INF/pages/account.jsp

```diff
@@ -68,19 +72,21 @@
 		<div class="container-fluid">
 			<%@ include file="./includes/header.jspf" %>
 			<%!
-			private String getUserAttribute(boolean isEditing, String attributeName, String attributeValue)
+			private String getUserAttribute(boolean isEditing, String attributeName, String attributeValue, String... type)

```

Comment:
can you change this to be an overloaded method instead of an vargs array as only the first value is used